src/modules/ejector/ejector.py

```diff
@@ -296,6 +295,12 @@ def _get_latest_exit_epoch(self, blockstamp: BlockStamp) -> tuple[EpochNumber, i
                 max_exit_epoch_number = val_exit_epoch
                 latest_to_exit_validators_count = 1
 
+        logger.info({
+            'msg': 'Calculate latest exit epoch',
```

Comment:
in the `_get_predicted_withdrawable_epoch` this value can be changed to 0, and this message will be misleading

suggest to put it later up in the `_get_predicted_withdrawable_epoch` function

Reply:
It's true, but `_get_predicted_withdrawable_epoch` called many time in cycle. And `_get_latest_exit_epoch` called once and actual for current blockchain state. We always can calculate `compute_exit_epoch`, because it is `ref_slot + some_constant`